internal/common/utils/random.go

```diff
@@ -0,0 +1,16 @@
+package utils
```

Comment:
Please indicate that these utils are intended to be used in tests by changing the pkg path to either `/internal/utils/test/` with defined aliased import `testutils` or simply change it to`/internal/testutils/`

Reply:
Decided to go with `/internal/utils/test` with defined aliased import `testutils`